src/rasterizer/rasterizer.py

```diff
@@ -0,0 +1,580 @@
+#!/bin/python3
+
+import argparse
+from time import time
+import numpy as np
+import matplotlib.pyplot as plt
+import torch
+
+
+def _sample_quadratic_bezier(control_points, steps):
+    '''
+    Sample a quadratic Bezier curve, given control points and steps.
+
+    Parameters
+    ----------
+    control_points : [3, 2]
+        The three control points of the Bezier curve.
+    steps : [num_steps,]
+        The steps to sample at. Usually a linspace.
+
+    Returns
+    -------
+    samples : [2, num_steps]
+    '''
+    P0 = control_points[0].unsqueeze(1)
+    P1 = control_points[1].unsqueeze(1)
+    P2 = control_points[2].unsqueeze(1)
+    samples = (P1 + (1 - steps)**2 * (P0 - P1) + steps**2 * (P2 - P1))
+    return samples
```

Comment:
I have a preference for an empty line before the return line. I thought yapf did as well, but it seems it doesn't.

Reply:
I don't think any formatter has a preference. [Black](https://github.com/ambv/black) also doesn't have a preference, for instance. In any case it's a simple enough function I think.